MeetHut.Backend/ClientApp/src/app/auth/auth.module.ts

```diff
@@ -12,6 +12,43 @@ import {
   SocialAuthServiceConfig,
   SocialLoginModule
 } from 'angularx-social-login';
+import { ParameterService } from '../shared/services';
+
+const socialConfigFactory = (parameterService: ParameterService) =>
+  new Promise((resolve) => {
+    parameterService.getAll().then((res) => {
+      const providers = [];
+
+      if (res) {
+        // Google
+        const googleClientIdParam = res.find(
+          (x) => x.key === 'Google.ClientId'
+        );
+        if (googleClientIdParam && googleClientIdParam.value) {
+          providers.push({
+            id: GoogleLoginProvider.PROVIDER_ID,
+            provider: new GoogleLoginProvider(googleClientIdParam.value)
+          });
+        }
+
+        // Microsoft
+        const msClientIdParam = res.find((x) => x.key === 'Microsoft.ClientId');
+        if (msClientIdParam && msClientIdParam.value) {
+          providers.push({
+            id: MicrosoftLoginProvider.PROVIDER_ID,
+            provider: new MicrosoftLoginProvider(msClientIdParam.value, {
+              redirect_uri: 'https://localhost:5001/signin-ms'
```

Comment:
Please add a parameter for it, or use the current host/port of the window

Reply:
@bencebaranyai So do I give back the current API address and port?

Reply:
1. use js (`window.location.hostname/protocol/port`...)
2. use config variable (static in appconfig)
3. ~~get it on asp side~~ (not really available since it can be accessed on "any" hostname)

Reply:
Ohh I see. I can't read. I misunderstood your first comment.